docs/product-assembly_migrating-from-BRMS-6.4-to-DM-7.0/src/main/asciidoc/code-troubleshooting-ref.adoc

```diff
@@ -1,58 +0,0 @@
-[id='code-troubleshooting-ref']
-= Troubleshooting build and execution errors in migrated Java projects
-Use the following information to resolve any build or execution errors after rebuilding migrated projects for {PRODUCT} {PRODUCT_VERSION}.
-
-== API changes in Drools modules
-
-The following Drools modules include some incompatible changes from BRMS 6.4 to {PRODUCT} {PRODUCT_VERSION}:
-
-* `kie-api`: the main API for all projects from KIE Group. The majority of changes are in the following classes:
-** `org.kie.api.task.*`
```

Comment:
Why the asterisk after some modules? Also, there is a period after several.

Reply:
It's wildcard. Like an ellipsis but for package names. i.e., "org.kie.api.task.  ..." Any package class names starting with that, and that may have a variety of endpoints.